OCPBUGS-49985:updates NAD JSON to include namespace (#89565)
openshift/openshift-docs: modules/nw-multus-create-network-apply.adoc

> * Install the OpenShift CLI (`oc`).
> * Log in as a user with `cluster-admin` privileges.
> * You are working in the namespace where the NAD is deployed.

Reviewer: ```suggestion
* You have installed the {oc-first}.
* You have logged in as a user with `cluster-admin` privileges.
* You have a network attachment definition (NAD) deployed in the namespace where you are working.
```
A few suggestions for prerequisites to maintain parallelism, expanding abbreviation, and using attribute.

> * Install the OpenShift CLI (`oc`).
> * Log in as a user with `cluster-admin` privileges.
> * You are working in the namespace where the NAD is deployed.

Reviewer: ```suggestion
* You have installed the {oc-first}.
* You have logged in as a user with `cluster-admin` privileges.
* You have a network attachment definition (NAD) deployed in the namespace where you are working.
```
A few suggestions for prerequisites to maintain parallelism, expanding abbreviation, and using attribute.

Reviewer 2: @shreyasiddhartha  A tiny nit here

"You are working in the namespace where you deployed the NAD."

Reviewer 2: Makes sense given the context of the proc.

    {
      "cniVersion": "0.3.1",
      "name": "work-network",
      "namespace": "namespace2", #<1>
      "type": "host-device",
      "device": "eth1",
      "ipam": {
        "type": "dhcp"
      }
    }
----

> <1> Optional: You can specify a namespace to which the NAD is applied. If you are working in the namespace where the NAD is to be deployed, this spec is not necessary.

Reviewer: ```suggestion
<1> Optional: Specify a namespace to apply the NAD. This spec is not necessary when working in the same namespace as the deployed NAD.
```
Suggestion to avoid passive voice.

Author: Sort of the same thing here as above.
